### PR TITLE
Changes on samplify backend to integrate with the switching endpoint

### PR DESCRIPTION
**Pull request by ssi-hu-janos-pelsoczi**
https://app.asana.com/0/1187510418082032/1186513356337911/f

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/researchnow/go-samplifyapi-client
 
-go 1.13
+go 1.14
```
Comment (sumitjawale) [inline go.mod:3]: Is this upgrade needed?

**Review by sumitjawale — COMMENTED** [review]
LGTM apart from the version upgrade. Not sure if it will impact any other stuff or something. Please check!

File: go.mod (hunk 1 above)
Comment (caseylucas) [reply go.mod:3]: Since this is a public repo, let's keep it 1.13

**Review by ashajyothi828 — APPROVED** [review]
Agree with comments on go version. Everything else looks good to me.

File: go.mod (hunk 1 above)
Comment (ssi-hu-janos-pelsoczi) [reply go.mod]: Done.

**Comment by ssi-hu-janos-pelsoczi** [comment]
Added SwichCompany endpoint

**Pull request merged**